bindings/nodejs/benchmark/index.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+const { suite, add, cycle, complete } = require('benny')
+const { Operator } = require('../index')
+const { S3Client, PutObjectCommand } = require('@aws-sdk/client-s3')
+const { Upload } = require('@aws-sdk/lib-storage')
+const crypto = require('crypto')
+
+const opendal = new Operator('s3', {
+  root: '/',
+  bucket: 'benchmark',
+  endpoint: process.env.AWS_S3_ENDPOINT,
+})
+
+const client = new S3Client({
+  endpoint: process.env.AWS_S3_ENDPOINT,
+  region: process.env.AWS_S3_REGION,
+  credentials: {
+    accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+    secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+  },
+  bucket: 'benchmark',
+})
+
+const s3 = {
+  write: (path, data) => {
+    const command = new PutObjectCommand({
+      Body: data,
+      Key: path,
+      Bucket: 'benchmark',
+    })
+    return client.send(command)
+  },
+  parallelUpload: (path, data) => {
```

Comment:
I would rather remove this case to maintain a simpler benchmark. Starting from a simple baseline is more beneficial for us. Additionally, I am particularly interested in the `read` performance as opposed to the `write` performance.